app/persistence/entity/lock-event/status.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package lock_event
+
+// TODO: Put statuses here
+const (
```

Comment:
If we are using the same statuses as `burn`, why don't we reuse the model?

Reply:
I am not yet sure what statuses will be used, that's why I preferred to play safe and create an additional file.